Package/Installer.cs

```diff
@@ -140,7 +152,18 @@ internal int UninstallThread()
 
         internal int RunCommand(string command, bool force, bool modifiesPackageFiles)
         {
-            var verb = System.Globalization.CultureInfo.CurrentCulture.TextInfo.ToTitleCase(command.ToLower()) + "ed";
+            string titlecase(string str) => System.Globalization.CultureInfo.CurrentCulture.TextInfo.ToTitleCase(str);
+            
+            var verb = titlecase(command.ToLower()) + "ed";
+            // Example usages:
+            // "Tried to {commandFriendlyName} {pkg.Name}, but there was nothing to do."
+            // "There was an error while trying to {commandFriendlyName} '{pkg.Name}'."
+            var commandFriendlyName = command; 
+            if (string.Equals(command, PrepareUninstall, StringComparison.OrdinalIgnoreCase))
```

Comment:
RunCommand call with two different commands and inside the call there is a whole special case for one of them. Can we clean this up a bit and just hardcode the strings?

![image](https://github.com/user-attachments/assets/0ece8d06-b841-4244-bea7-a52d921ef7a3)